webapp/src/widgets/SpeechToText/types.ts

```diff
@@ -31,3 +31,13 @@ export interface DeepgramResponse {
     }
   }
 }
+
+export enum SpeechModel {
+  nova2 = 'nova-2-ea',
```

Comment:
Do they have built-in multilingual support? i.e. able to handle one sentence mixing two languages, and able to handle user switching languages between Q&As